nemo_skills/prompt/config/judge/code.yaml

```diff
@@ -0,0 +1,82 @@
+system: ""
+
+# Defines the judge prompt for code generation tasks
+# You need to use keys you want to use from the input dict using {}
+# For example, you generally want {instruction} and {output} keys for genetic instruct
+# Allowed keys for judgement - {Yes or No}, {Yes}, {No}. Other keys must be explicitly escaped in code.
+user: |
+  '''
+  You are an expert python programmer. 
+
+  Below is a question and code solution. Decide if the solution follows the below criteria and give a final Yes/No, and place it in the <judge></judge> tags.
+  
+  Only look at the function generated, not any examples/print statements etc. Just the core logic.
+  
+  Please first briefly describe your reasoning (in less than 30 words) without writing any code, and then write Decision: \\\\boxed{Yes or No} in your last line.
```

Comment:
does this prompt need some special logic for .fill? Otherwise, it should complain as you don't escape with {{}} here?

Reply:
Good point, our code actually does have that special logic. Does wrapping it with {{..}} fix this problem? Ill update it then.

Reply:
Yes, if you properly wrap {} and escape \ it should work by default

We might consider changing our default placeholders to use {{}} and then the normal {} can be used whenever applicable (e.g. in json format, etc.). Let me know if you guys have a bunch of use-cases like this, would need to update all the prompts, but might be more future proof

Reply:
Its actually quite rare to have {} inside the prompt. Id rather follow the json prompting schema and then special cases like these use the workaround with {{}}